SQL/0000-00-00-schema.sql

```diff
@@ -555,6 +555,7 @@ CREATE TABLE `flag` (
   `Flag_status` enum('P','Y','N','F') default NULL,
   `UserID` varchar(255) default NULL,
   `Testdate` timestamp NOT NULL default CURRENT_TIMESTAMP on update CURRENT_TIMESTAMP,
+  `Data` TEXT default NULL,
```

Comment:
No, we can't :(

There's bottlenecks in both our production servers (which aren't on 5.7), and MariaDB (which doesn't have the JSON data type and won't until 10.3.)